pkgs/tools/misc/lazydocker/default.nix

```diff
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "lazydocker";
-  version = "0.7";
+  version = "0.7.4";
 
   src = fetchFromGitHub {
     owner = "jesseduffield";
     repo = "lazydocker";
     rev = "v${version}";
-    sha256 = "0vai88g31yf55988paqzs7fqlxgi0ydrsgszzjig9ai3x9c52xim";
+    sha256 = "03l6gs4p9p8g0ai6wqg9024rp0pd13m0b9y3sy1ww5afwxb82br6";
   };
 
-  modSha256 = "1iin1m6s9xxdskvj6jy2jwlqrsrm432ld13cpa28hpx7pylx61ij";
+  modSha256 = "16idjsh0mlh2xra434ywvgy1l9wzhgmbc5kx32b8jzx7w7z9i78z";
```

Comment:
```suggestion
  modSha256 = "1hzrin8dfsfnxpc37szc1449s235w0dr24albswz06fjnl4bbs5y";
```

Reply:
Not sure how that happened :thinking: Thanks for pointing it out!